gocourse05/camera/dayCamera_test.go

```diff
@@ -0,0 +1,50 @@
+package camera
+
+import (
+	"testing"
+)
+
+func TestDayCameraRetrieveData(t *testing.T) {
+	t.Run("Found", func(t *testing.T) {
+		c := NewDayCamera(1, "Canon", []Data{NewDayCameraData(1, "Коза", "Побігла вліво")})
+
+		data, _ := c.retrieveData()
```

Comment:
Завжди error треба хендлити.
```suggestion
		data, err := c.retrieveData()
```

Уяви, що хтось перерефакторив `retrieveData` і error тепер став не `nil`. Як тест це задетектить, якщо ігнорити `err`?